crates/driver/example.toml

```diff
@@ -1,15 +1,15 @@
 [[solver]]
 name = "mysolver" # Arbitrary name given to this solver, must be unique
-endpoint = "http://0.0.0.0:33219/solve"
+endpoint = "http://0.0.0.0:7872"
 absolute-slippage = "12" # Denominated in wei, optional
```

Comment:
Can we change this to a meaningful example? I saw in the `infrastructure` repo that we actually used this nonsensical value for a bit (probably just because it was copied from the example without much thought).

Reply:
> Can we change this to a meaningful example?

This address would work with a `solvers` binary run locally with the default arguments which seems sensible to me.
What else do you have in mind given the fact that the `driver` run locally will never just work out of the box with any address?

Reply:
It turns out the `solvers` and `driver` took the `absolute-slippage` in different units.
I changed the `solvers` to expect `U256` now (same as driver) because with that deploying on chains where the base token does not have 18 decimals is easily handled.